libs/game-state/src/lib/services/cards/shallow-gravedigger.ts

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable no-mixed-spaces-and-tabs */
+import { CardIds, CardType, GameTag, hasMechanic } from '@firestone-hs/reference-data';
+import { canBeDiscoveredByClass, hasCorrectType } from '../../related-cards/dynamic-pools';
+import { StaticGeneratingCard, StaticGeneratingCardInput } from './_card.type';
+import { filterCards } from './utils';
+
+export const ShallowGravedigger: StaticGeneratingCard = {
```

Comment:
It adds a card to the hand, so it should use the GeneratingCard interface as well